docs/modules/install/pages/manual.adoc

```diff
@@ -121,6 +121,11 @@ bin/rails db:create db:migrate
 bin/rails db:seed
 ----
 
+[NOTE]
+====
+Please refer to xref:install:noseed.adoc[No-seed Installation] section if you want to setup your instance with no seeds
```

Comment:
I think seeds is too technical/rails specific and probably something that isn't familiar with programming wouldn't know what's that. How about renaming this to "empty installation" or something like that instead of "no-seed installation" ? 


```suggestion
Please refer to xref:install:empty-database.adoc[Empty database installation] section if you want to setup your instance on an empty database (without any seeds)
```

Reply:
Good point. I was thinking of the same issue.